docs/how-to-articles/how-to-simulate-networks-with-netem.md

```diff
@@ -0,0 +1,120 @@
+# Using NetEm to emulate networks
```

Comment:
All headings should be written with capital letters. Change "Using NetEm to emulate networks" to "Using NetEm to Emulate Networks". Please look through the whole document and correct other heading as well.